src/main/kotlin/blackjack/model/Participant.kt

```diff
@@ -0,0 +1,37 @@
+package blackjack.model
+
+interface Participant {
+    val name: String
+    var isActive: Boolean
+    val cardsInHand: MutableList<Card>
```

Comment:
Since `Participant` can evaluate total value of cards by itself,
I think `var isActive` is unnecessary since we can find out if they are active or not right away.
By having this `var isActive`, some other developer might change this by accident, creating a bug.
I'd like to know how you think about this :)